src/gui/src/components/explorer-grid/selected-item/item-header.tsx

```diff
@@ -108,84 +118,195 @@ const Downloads = () => {
+          )}
+        />
+        {selectedItemDetails.downloads && (
+          <div className="col-span-3 flex h-full cursor-default items-center justify-center text-center">
+            <p className="w-full text-sm font-medium text-foreground">
+              {selectedItemDetails.downloads.weekly.toLocaleString()}{' '}
+              <span className="text-muted-foreground">Downloads</span>
+            </p>
+          </div>
+        )}
       </div>
       <div
         className={cn(
-          'px-6 pt-3',
-          insights ? 'border-t-[1px] border-muted' : 'hidden',
+          'grid w-full grid-cols-12',
+          hasMetadata ?
+            'border-b-[1px] border-t-[1px] border-muted'
+          : 'hidden',
         )}>
-        <PackageInsights />
+        <PackageMetadata
+          className={cn(
+            'col-span-9 border-r-[1px] border-muted pl-6',
+            hasMetadata && 'py-3',
+          )}
+        />
+        <PackageOverallScore className="col-span-3" />
       </div>
     </motion.div>
   )
 }
 
-const PackageInsights = () => {
-  const { insights } = useSelectedItem()
+const PackageMetadata = ({ className }: { className?: string }) => {
+  const { manifest, unpackedSize, selectedItemDetails } =
+    useSelectedItem()
+  const tarballUrl = selectedItemDetails.versions?.[0]?.tarball
+  const INLINE_CODE_STYLES =
+    'text-muted-foreground cursor-default relative mx-0 inline-flex items-center font-inter font-medium tracking-wide'
 
-  if (!insights || insights.length === 0) return null
+  if (!manifest) return null
+
+  const LICENSE_TYPES = [
+    'MIT',
+    'ISC',
+    'Apache-2.0',
+    'GPL-3.0',
+    'GPL-2.0',
+    'LGPL-3.0',
+    'BSD-2-Clause',
+    'BSD-3-Clause',
+    'MPL-2.0',
+    'Unlicense',
+    'CC0-1.0',
+  ]
 
   return (
-    <div className="flex w-full flex-wrap gap-3">
-      {insights.map((insight, idx) => (
-        <InsightBadge
-          key={idx}
-          color={getAlertColor(insight.severity)}
-          tooltipContent={`${insight.severity} severity`}>
-          {insight.selector}
-        </InsightBadge>
-      ))}
+    <div className={cn('flex w-full gap-2', className)}>
+      {manifest.private && (
+        <InlineCode
+          variant="unstyled"
+          className={cn(
+            INLINE_CODE_STYLES,
+            'gap-1 bg-rose-500/20 px-2 text-rose-600 dark:bg-rose-500/20 dark:text-rose-400',
+          )}>
+          <EyeOff size={16} className="mb-0.5" />
+          Private
+        </InlineCode>
+      )}
+      {manifest.license &&
+        LICENSE_TYPES.includes(manifest.license) && (
+          <InlineCode
+            variant="mono"
+            className={cn(INLINE_CODE_STYLES, 'gap-1 px-2')}>
+            <Scale size={16} className="mb-0.5" />
+            {manifest.license}
+          </InlineCode>
+        )}
+      {manifest.type && (
+        <InlineCode variant="mono" className={cn(INLINE_CODE_STYLES)}>
+          {manifest.type}
```

Comment:
I'd prefer printing `ESM` or `CJS`, it's also shorter!

```suggestion
          {manifest.type === 'module' : 'ESM' : 'CJS'}
```